brooklyn-docs/guide/ops/externalized-configuration.md

```diff
@@ -122,6 +122,23 @@ brooklyn.location.jclouds.aws-ec2.credential=$brooklyn:external("mysupplier", "a
 {% endhighlight %}
 
 
+## Referring to External Configuration in catalog items
```

Comment:
Capitalisation of "catalog items" is inconsistent with other section headings. /nitpick